src/meta_schedule/postproc/rewrite_unbound_block.cc

```diff
@@ -195,17 +200,31 @@ bool RewriteUnboundBlockNode::Apply(const tir::Schedule& sch) {
     if (bind_type == tir::BindType::kBindBlock) {
       sch->Bind(fused, "blockIdx.x");
     } else if (bind_type == tir::BindType::kBindBlockThread) {
-      Array<LoopRV> splits = sch->Split(fused, {NullOpt, Integer(this->warp_size_)});
-      ICHECK_EQ(splits.size(), 2);
-      sch->Bind(splits[0], "blockIdx.x");
-      sch->Bind(splits[1], "threadIdx.x");
+      int extent_size = 0, max_block = 256;
```

Comment:
Note that 256 is is a magic number for `max_block`. Ideally we should:
- Rename `max_block` => `max_threadblock`
- Make it a parameter to the constructor of `RewriteUnboundBlock`